home/tests/test_content_import_export.py

```diff
@@ -847,6 +847,42 @@ def test_cpi_validation_failure_update(self, csv_impexp: ImportExport) -> None:
             == "Validation error: {'translation_key': ['“BADUUID” is not a valid UUID.']}"
         )
 
+    def test_ContentPageIndex_required_fields(self, csv_impexp: ImportExport) -> None:
+        """
+        Importing an CSV file with only the required fields shoud not break
+
+        """
+
+        csv_bytes = csv_impexp.import_file("contentpage_index_required_fields.csv")
+        content = csv_impexp.export_content()
+        src, dst = csv_impexp.csvs2dicts(csv_bytes, content)
+
+        content_pages = ContentPageIndex.objects.all()
+
+        for page_data, content_page in zip(src, content_pages, strict=False):
+            assert page_data.get("slug", "") == content_page.slug
+
+        assert content_pages.count() == len(list(src))
+
+    def test_ContentPage_required_fields(self, csv_impexp: ImportExport) -> None:
+        """
+        Importing an CSV file with only the required fields shoud not break
+
+        """
+
+        csv_bytes = csv_impexp.import_file("contentpage_required_fields.csv")
+        content = csv_impexp.export_content()
+        src, dst = csv_impexp.csvs2dicts(csv_bytes, content)
+
+        content_pages = list(ContentPageIndex.objects.all()) + list(
+            ContentPage.objects.all()
+        )
```

Comment:
As with the previous test, we can be a lot more explicit:

```python
[main_menu] = ContentPageIndex.objects.all()
[first_time_user, health_info, self_help] = ContentPage.objects.all()
# And then a slug assertion for each page.
```

Reply:
Got it, I kept the line that defines `content_pages` so checking the sum a a bit cleaner.

Reply:
Do we really need to assert on the sum if we already get an exception when any of the queries returns the wrong number of results?

Reply:
Correct, no need for that check, removed.